src/test/unit/sportsradar/extract/gamefeeds.py

```diff
@@ -54,3 +54,54 @@ def test_game_feeds_with_game_statistics(mock_data_store, mock_getenv):
                                         game_id=game_id, file_format=TestConstants.FORMAT,
                                         api_key=TestConstants.API_KEY)
     assert result.status_code == 200
+
```

Comment:
please apply new unit tests updated on the main branch.

Reply:
Changes Implemented